src/components/toggle/theme.css

```diff
@@ -29,10 +29,11 @@
     overflow: hidden;
     opacity: 0;
     margin: 0;
+    position: absolute;
```

Comment:
Q: shouldn't the toggle be `position: relative` then?

Reply:
input is hidden. I am setting it to `absolute` so it wont mess with `justify-content: space-between`

Reply:
I see